hwstats/backend.py

```diff
@@ -90,3 +90,16 @@ def query_memory_percent_with_time(pid_hash: str, session: Session) -> list[tupl
         .all()
     )
     return result
+
+
+def query_Disk_percent_with_time(pid_hash: str, session: Session) -> list[tuple[datetime, float]]:
```

Comment:
The type signature at the end needs an update since this will return a list of three element tuples: `list[tuple[datetime, float, float]]`

Reply:
I'd also suggest we name the query to indicate that its getting reads and writes like: `query_disk_read_write_with_time